src/controllers/controller.line.js

```diff
@@ -210,17 +208,17 @@ module.exports = DatasetController.extend({
 			key = keys[i];
 			values[key] = resolve([
 				custom[key],
-				dataset[key],
+				datasetOpts[key],
 				elementOptions[key]
 			], context);
 		}
 
 		// The default behavior of lines is to break at null values, according
 		// to https://github.com/chartjs/Chart.js/issues/2435#issuecomment-216718158
 		// This option gives lines the ability to span gaps
-		values.spanGaps = valueOrDefault(dataset.spanGaps, options.spanGaps);
-		values.tension = valueOrDefault(dataset.lineTension, elementOptions.tension);
-		values.steppedLine = resolve([custom.steppedLine, dataset.steppedLine, elementOptions.stepped]);
+		values.spanGaps = valueOrDefault(datasetOpts.spanGaps, options.spanGaps);
+		values.tension = valueOrDefault(datasetOpts.tension, elementOptions.tension);
```

Comment:
`lineTension` was changed to `tension`, I don't recall and didn't quickly find any discussion about this. Accidental or intentional?

Reply:
oof. that's bad. yeah, that was accidental. thank you for catching that!  I've fixed that now